src/ol/control/ScaleLine.js

```diff
@@ -179,7 +179,7 @@ ScaleLine.prototype.updateElement_ = function() {
     Units.METERS;
   let pointResolution =
       getPointResolution(projection, viewState.resolution, center, pointResolutionUnits);
-  if (projection.getUnits() != Units.DEGREES && units == ScaleLineUnits.METRIC) {
+  if (projection.getUnits() != Units.DEGREES) {
```

Comment:
You'll probably need
```js
if (projection.getUnits() != Units.DEGREES && units != ScaleLineUnits.DEGREES) {
```

Reply:
I think the `projection.getUnits()` 
part needs to be 
```
includes([Units.FEET, Units.METERS, Units.USFEET], projection.getUnits())
```

and `pointResolutionUnits == Units.METERS` or what you wrote (maybe this one is better documenting that we are converting from meters).

Reply:
Please add a test case that shows what you're trying to achieve. It seems to me you're confusing projection units and scale line units. Also please reference the issue that this pull request is supposed to fix, for future reference.